longevity_test.py

```diff
@@ -306,15 +306,15 @@ def test_user_batch_custom_time(self):
 
         self.db_cluster.add_nemesis(nemesis=self.get_nemesis_class(), tester_obj=self)
 
-        batch_size = self.params.get('batch_size', default=1)
+        batch_size = self.params.get('batch_size')
 
-        if not self.params.get('reuse_cluster', default=False):
+        if not self.params.get('reuse_cluster'):
             self._pre_create_templated_user_schema()
 
             # Start new nodes
             # we are starting this test case with only one db to make creating of the tables quicker
             # gossip with multiple node cluster make this painfully slower
-            add_node_cnt = self.params.get('add_node_cnt', default=1)
+            add_node_cnt = self.params.get('add_node_cnt') or 1
```

Comment:
add to `test_defaults.yaml`, no need to keep the default value here

Reply:
done